project/Testing.scala

```diff
@@ -8,31 +8,44 @@ object Testing {
   lazy val NoDockerTest = config("nodocker") extend Test
   lazy val CromwellIntegrationTest = config("integration") extend Test
   lazy val CromwellNoIntegrationTest = config("nointegration") extend Test
+  lazy val DbmsTest = config("dbms") extend Test
+
+  lazy val DockerTestTag = "DockerTest"
+  lazy val UseDockerTaggedTests = Tests.Argument("-n", DockerTestTag)
+  lazy val DontUseDockerTaggedTests = Tests.Argument("-l", DockerTestTag)
+
+  lazy val CromwellIntegrationTestTag = "CromwellIntegrationTest"
+  lazy val UseCromwellIntegrationTaggedTests = Tests.Argument("-n", CromwellIntegrationTestTag)
+  lazy val DontUseCromwellIntegrationTaggedTests = Tests.Argument("-l", CromwellIntegrationTestTag)
+
+  lazy val DbmsTestTag = "DbmsTest"
+  lazy val UseDbmsTaggedTests = Tests.Argument("-n", DbmsTestTag)
+  lazy val DontUseDbmsTaggedTests = Tests.Argument("-l", DbmsTestTag)
 
   /*
   The arguments that will be added to the default test config, but removed from all other configs.
   `sbt coverage test` adds other arguments added to generate the coverage reports.
   Tracking the arguments we add to the default allows one to later remove them when building up other configurations.
  */
-  lazy val defaultTestArgs = Seq(Tests.Argument("-l", "DockerTest"), Tests.Argument("-l", "CromwellIntegrationTest"))
+  lazy val defaultTestArgs = Seq(DontUseDockerTaggedTests, DontUseCromwellIntegrationTaggedTests, DontUseDbmsTaggedTests)
 
   val testSettings = List(
-    // `test` (or `assembly`) - Run all tests, except docker and integration
+    // `test` (or `assembly`) - Run all tests, except docker and integration and DBMS
     testOptions in Test ++= defaultTestArgs,
     // `alltests:test` - Run all tests
     testOptions in AllTests := (testOptions in Test).value.diff(defaultTestArgs),
-    // `docker:test` - Run docker tests, except integration
-    testOptions in DockerTest := (testOptions in Test).value.diff(defaultTestArgs) ++
-      Seq(Tests.Argument("-n", "DockerTest"), Tests.Argument("-l", "CromwellIntegrationTest")),
+    // `notests:test` - Run no tests
```

Comment:
I can't remember if you said you were going to change/remove this because `notests:test` is confusing...